### ISSUE TITLE
devicon lookup fails when using ansi colors

### ISSUE THREAD

**Issue by petobens** [coreyja/devicon-lookup#10]
Hi! I recently noticed the following behaviour:
```
fd . | devicon-lookup | fzf
```
shows correct icons whereas
```
fd --color=always . | devicon-lookup | fzf --ansi
```
doesn't:

![image](https://user-images.githubusercontent.com/2583971/54732742-27f26f80-4b74-11e9-89c9-221da420287a.png)

**Comment by coreyja** [comment]
Hey again @petobens! I bet this is caused by not handling/stripping the ansi color codes and therefor looking for an extension with the extra color code at the end.

I wonder if implementing #7 would help have some of the internals required for this 🤔

**Comment by coreyja** [comment]
This crate might also come in handy for stripping ansi codes for looking up the extension https://github.com/mackwic/colored

**Comment by coreyja** [comment]
@petobens This is implemented and released in version 0.6.0! You have to provide the `--color` option to activate it, here is a screenshot example

<img width="427" alt="Screen Shot 2019-03-24 at 8 23 21 PM" src="https://user-images.githubusercontent.com/964771/54888335-cd5a5b80-4e72-11e9-8bb3-0a2c4272a620.png">

You should be able to do something like this following now

```
fd --color=always . | devicon-lookup -c | fzf --ansi
```

`-c` is the shorthand for `--color`

**Comment by petobens** [comment]
Works great. Thanks!!
